testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/VirtualHostTestCase.java

```diff
@@ -17,10 +17,10 @@
  * You should have received a copy of the GNU Lesser General Public
  * License along with this software; if not, write to the Free
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
- * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ * 2110-1301 USA, or see the FSF site: http://www.fsf.org.
```

Comment:
Is it correct to drop the leading 0 from the phone number?

Reply:
That's the postal code so it should probably not be changed.

Reply:
LOL yes ;-)